### DISCUSSION TITLE
［求助］早上起来被saber劈了

### DISCUSSION THREAD

**Discussion by lbghhj** [share121/inter-knot#502] [其他]
![1723048240612](https://github.com/user-attachments/assets/4e80061b-f7aa-44e7-a12d-0a4021620325)
这么大个咖喱棒，我会不会s啊啊

**Comment by xinxingjie** [comment]
我不能苟同你的观点，我认为咖喱棒就应该在早上拌24号混凝土搭配德马拉电池2型在光映广场的HDD系统里喝慢汀特调

**Comment by ZHMcole** [comment]
没事，体育生一声都不吭......吧？

**Comment by light814** [comment]
这个委托我接了!

**Comment by XiaoFffeng** [comment]
如果你死了，那现在是谁在发帖？
